<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paintMainWindow</class>
 <widget class="QMainWindow" name="paintMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Paint</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="m_graphLayout" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="6">
        <widget class="QPushButton" name="m_PB_Text">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/rsc/rsc/text-font.png</normaloff>:/rsc/rsc/text-font.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="m_PB_Rect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Shapes</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/rsc/rsc/shapes.png</normaloff>:/rsc/rsc/shapes.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="m_PB_Pencil">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Pencil</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/rsc/rsc/pencil.png</normaloff>:/rsc/rsc/pencil.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="m_PB_Color">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/rsc/rsc/color-palette.png</normaloff>:/rsc/rsc/color-palette.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="m_PB_Eraser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Eraser</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/rsc/rsc/erase.png</normaloff>:/rsc/rsc/erase.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="m_PB_Brush">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Brush</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/rsc/rsc/brush.png</normaloff>:/rsc/rsc/brush.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="6">
        <widget class="QGraphicsView" name="m_graphScene">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>165577</width>
           <height>165577</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="interactive">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="dragMode">
          <enum>QGraphicsView::NoDrag</enum>
         </property>
         <property name="transformationAnchor">
          <enum>QGraphicsView::NoAnchor</enum>
         </property>
         <property name="viewportUpdateMode">
          <enum>QGraphicsView::MinimalViewportUpdate</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
